go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/ExpediaGroup/flyte-slack
 go 1.14
 
 require (
-	github.com/HotelsDotCom/flyte-client v0.0.0-20191018093418-d4ecefe9ad35
+	github.com/HotelsDotCom/flyte-client v1.0.0
 	github.com/HotelsDotCom/go-logger v0.0.0-20180518131502-802095993e48
-	github.com/slack-go/slack v0.6.2
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/slack-go/slack v0.9.1
```

Comment:
This is quite big change from 0.6.2 > 0.9.1
I don't know what the changes were in between and if there were any breaking changes. I understand this has been fully tested

Reply:
confirmed, the way how things were tested is by making a instance of updated flyte-slack pack and registring it on API server, then existing commands were tested with new GetChannelInfo on Slack Sandbox instance